Documentation/gettingstarted/policy-creation.rst

```diff
@@ -0,0 +1,81 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _policy_verdicts:
+
+*******************************
+Creating policies from verdicts
+*******************************
+
+Policy Audit mode allows the user allow all traffic to pass through Cilium
+while logging all connections that would otherwise be dropped by policy using
+policy verdict notifications. Policy audit mode may be configured for the
+entire daemon using ``--policy-audit-mode=true``. Layer 3 and 4 policies are
+currently supported. When policy audit mode is enabled and you've installed a
+policy that doesn't match traffic, ``cilium monitor`` prints the following:
+
+.. code-block:: shell-session
+
+   # cilium monitor -t policy-verdict
+   ...
+   Policy verdict log: flow 0x63113709 local EP ID 1121, remote ID 2986, dst port 80, proto 6, ingress true, action audit, match none, 10.29.210.187:54134 -> 10.29.50.40:80 tcp SYN
```

Comment:
Once #12137 is merged, this example output should probably be updated to the new format.